src/lib/params-serializer.test.ts

```diff
@@ -0,0 +1,106 @@
+import test from 'ava'
+
+import {
+  paramsSerializer,
+  UnserializableParamError,
+} from './params-serializer.js'
+
+test('serializes empty object', (t) => {
+  t.is(paramsSerializer({}), '')
+})
+
```

Comment:
Should we probably add test cases for checking params as null or undefined?

Reply:
This function implements the axios paramsSerializer which only takes `Record` type, so if params is null or undefined this function should not get called.